content/en/account_management/api-app-keys.md

```diff
@@ -24,18 +24,27 @@ The recommended best practice for scoping application keys is to grant your keys
 
 **Notes:**
 
-- Users or service accounts with [permissions][4] to create or edit application keys can scope application keys. A user must have the `user_app_keys` permission to scope their own application keys, or the `org_app_keys_write` permission to scope application keys owned by any user in their organization. A user must have the `service_account_write` permission to scope application keys for service accounts.
+- Users or service accounts with [permissions][3] to create or edit application keys can scope application keys. A user must have the `user_app_keys` permission to scope their own application keys, or the `org_app_keys_write` permission to scope application keys owned by any user in their organization. A user must have the `service_account_write` permission to scope application keys for service accounts.
 - Application owners cannot authorize an application if they are missing any required permissions, even if they scope an application key with authorization scopes that they do not have.
-- Errors due to missing permissions when writing application keys or authorizing applications will display a `403 Forbidden` error. More information about various error responses can be found in the [Datadog API][5] documentation.
+- Errors due to missing permissions when writing application keys or authorizing applications will display a `403 Forbidden` error. More information about various error responses can be found in the [Datadog API][4] documentation.
 - If a user's role or permissions change, authorization scopes specified for their application keys remain unchanged.
 
+### Actions API access
+
+The Actions Platform APIs include:
+- [App Builder][5]
+- [Actions Connections][6]
+- [Workflow Automation][7]
+
+Each API requires specific permissions for individual endpoints. Because these APIs often work together, there is a simplified scoping process through the **Actions API Access** feature. Enabling **Actions API Access** adds all the required permissions for the Actions Platform APIs. 
```

Comment:
Sorry I didn't ask about this earlier when we discussed your work! What is **Actions API Access**? Is it a permission? If not, how do users enable it? It sounds like a permission. Elsewhere in this doc, we assume that people already know how to set permissions.